pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -3869,7 +3869,8 @@ SELECT ‹'yes'› IN (‹'no'›, ‹'maybe'›, ‹'yes'›)
 query T
```

Comment:
Why don't these bad boys need a rowsort?

Reply:
It's a single row - just looks like two lines in some editors due to the carriage return.